packages/browser/package.json

```diff
@@ -42,7 +42,6 @@
     "playwright": "^1.17.1",
     "rollup": "^1.10.1",
     "rollup-plugin-commonjs": "^9.3.4",
-    "rollup-plugin-license": "^0.8.1",
     "rollup-plugin-node-resolve": "^4.2.3",
     "rollup-plugin-terser": "^4.0.4",
```

Comment:
Can we hoist the rest of the roll-up plugins as well?

Reply:
Working on it! 🙂

(Really what I'm working on is updating them one by one and making sure nothing important changes. I'll hoist each one as I update it.)

Reply:
Nice!